docs/new-concept-users-and-groups.md

```diff
@@ -115,6 +115,16 @@ Bindings can be managed via the meshStack UI or API. When a user or group is bou
 
 You can configure meshStack to require 4 eye principle for workspace and project permissions. This configuration enforces that at least two workspace owners or managers have to approve access before a user binding is created or modified.
 
+## User and Role Bindings
+
+User and role bindings define how users and groups are assigned specific roles within workspaces and projects. These bindings control access and permissions, ensuring that users have the appropriate level of responsibility for their tasks.
+
+Bindings can be managed via the meshStack UI or API. When a user or group is bound to a role, they inherit the permissions associated with that role for the relevant workspace or project.
+
+## 4 Eye Principle
+
+You can configure meshStack to require 4 eye principle for workspace and project permissions. This configuration enforces that at least two workspace owners or managers have to approve access before a user binding is created or modified.
```

Comment:
Wasn't that also on another page? Can you link there from here?

Reply:
Also in general I'm not so sure how well-settled the name "4 eye principle" is. Maybe but in brackets (Approvals) and mention the word Approvals in this section

Reply:
Good point

Reply:
your read that in the dedicated guide